config/initializers/simple_form_bootstrap.rb

```diff
@@ -0,0 +1,216 @@
+require 'simple_form_extensions'
```

Comment:
I spent way too long trying to decipher all these options before I realized that this file was automatically generated :-P.